Revert z/m fields description, as a 3.10 feature (#4667)
qgis/QGIS-Documentation: source/docs/user_manual/managing_data_source/opening_data.rst

Importing a delimited text file
-------------------------------

> Delimited text file (e.g. :file:`.csv`, :file:`.txt`) can be loaded in QGIS

Reviewer: ```suggestion
A delimited text file (e.g. :file:`.csv`, :file:`.txt`) can be loaded in QGIS
```

> using the tools described above. However, loaded this way, it'll show up like a

Reviewer: ```suggestion
using the tools described above. But when it is loaded this way, it will show up as a
```

> simple table data. Sometimes, delimited text files can contain geometric data

Reviewer: ```suggestion
simple table. Sometimes, delimited text files can contain geometry data
```

> you'd want to visualize; this is what the |addDelimitedTextLayer|:guilabel:`Add

Reviewer: ```suggestion
that QGIS can utilize; this is what the |addDelimitedTextLayer|:guilabel:`Add
```
